src/Plugin/Field/FieldFormatter/FileAudioFormatter.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+ * @file
+ *  Contains Drupal\file_entity\Plugin\Field\FieldFormatter\FileAudioFormatter
+ */
+
+namespace Drupal\file_entity\Plugin\Field\FieldFormatter;
+
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Template\Attribute;
+use Drupal\file\Plugin\Field\FieldFormatter\FileFormatterBase;
+
+/**
+ * Plugin implementation of the 'file_audio' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "file_audio",
+ *   label = @Translation("Audio"),
+ *   field_types = {
+ *     "file"
+ *   }
+ * )
+ */
+class FileAudioFormatter extends FileFormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultSettings() {
+    return array(
+      'controls' => TRUE,
```

Comment:
we will need config schema for those settings, should be relatively easy. Some tests would be great, then that would be enforced.